src/views/Home/components/Prizes/index.view.tsx

```diff
@@ -1,8 +1,72 @@
 import React from "react";
```

Comment:
#71 allows us to remove this statement completely if we do not use react within a file; however, since we use it here we can simply do: `import * as React from "react";`. With newer versions of react and typescript coming out, if react is not used within a file, you don't have to import react. It's pretty neat stuff!